src/Routing/UrlGenerator.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Laravel\Lumen\Routing;
 
-use Laravel\Lumen\Application;
+use Illuminate\Contracts\Foundation\Application;
```

Comment:
`Laravel\Lumen\Application` in 5.2 is no longer implementing `Illuminate\Contracts\Foundation\Application`.